src/main/resources/assets/tconstruct/book/materials_and_you/sections/tools.json

```diff
@@ -1,32 +1,37 @@
 [
   {
-    "name": "repairing",
+	"name": "terminology",
```

Comment:
Why is the modifier terminology page in the tools section? Why not in the modifiers section?